tests/test_segmentation_islands_effect.py

```diff
@@ -0,0 +1,94 @@
+from pathlib import Path
+
+import numpy as np
+import pytest
+from undo_stack import UndoStack
+
+from trame_slicer.core import IOManager
+from trame_slicer.segmentation import SegmentationEffectIslands
+
+
+@pytest.fixture
+def undo_stack(a_segmentation_editor):
+    undo_stack = UndoStack()
+    a_segmentation_editor.set_undo_stack(undo_stack)
+    return undo_stack
+
+
+@pytest.fixture
+def an_io_manager(a_slicer_app):
```

Comment:
You don't need to create a new IO manager as it is already present in the `a_slicer_app` fixture.

If you want a shortcut, you can use : 

```python
@pytest.fixture
def an_io_manager(a_slicer_app):
    return a_slicer_app.io_manager
    
 ```